airbyte-integrations/connectors/destination-dev-null/README.md

```diff
@@ -1,6 +1,6 @@
-# Destination Dev Null
```

Comment:
Ignore this, I should just pick up whatever @stephane-airbyte did for the readme